services/twilio_verify.js

```diff
@@ -0,0 +1,51 @@
+const twilio = require('twilio');
```

Comment:
Do any of the `twilio` methods called here throw exceptions we need to wrap before returning REST responses? If so it might be worthwhile to pass `twilio` as a constructor parameter so that we can mock failures in the test suite.

Reply:
None worth handling, only specific errors for invalid/missing arguments:
https://github.com/twilio/twilio-node/blob/main/lib/rest/verify/v2/service/verificationCheck.js#L76-L81
https://github.com/twilio/twilio-node/blob/main/lib/rest/verify/v2/service/verification.js#L93-L101

Reply:
Actually i'm wrong - looks like there is a max attempts error that we should cover, and probably more. I'll have a deeper look in to this 👀

Reply:
Updated to handle errors we're likely to encounter, and as you suggested passing `twilio` as an argument so it could be tested :)